### PR TITLE
added resource activity

### PR DESCRIPTION
**Pull request by MadelynTav**

### CODE REVIEW

File: app/src/main/java/boston/hackingforgood/resourceapp/components/home/HomeActivity.java [hunk 1]
```diff
@@ -1,15 +1,25 @@
...
+    @Bind(R.id.viewPager)
+    ViewPager viewPager;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_main);
+        setContentView(R.layout.activity_home);
+        ButterKnife.bind(this);
+        FragmentManager fragmentManager = getSupportFragmentManager();
```
Comment (hdavidzhu) [inline app/src/main/java/boston/hackingforgood/resourceapp/components/home/HomeActivity.java:25]: move to separate method called `configure...` so it's outside of the lifecycle

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -18,7 +18,7 @@
         </activity>
         <activity android:name=".components.about.AboutActivity">
         </activity>
-        <activity android:name=".components.signup.SignupActivity">
+        <activity android:name=".components.signup.SignUpActivity">
```
Comment (hdavidzhu) [inline app/src/main/AndroidManifest.xml]: if this is SignUp, you can change the package name to `signUp`

File: app/src/main/java/boston/hackingforgood/resourceapp/components/home/HomeActivityFragment.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+/**
+ * A simple {@link Fragment} subclass.
+ */
+public class HomeActivityFragment extends Fragment {
+    @Bind(R.id.recyclerView)
+    RecyclerView recyclerView;
+
+
+    public HomeActivityFragment() {
+        // Required empty public constructor
```
Comment (hdavidzhu) [inline app/src/main/java/boston/hackingforgood/resourceapp/components/home/HomeActivityFragment.java]: is this really required?

File: app/src/main/java/boston/hackingforgood/resourceapp/components/home/HomeActivityFragment.java [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.fragment_home_activity, container, false);
+        ButterKnife.bind(this, view);
+
+        List<Object> objects = new ArrayList<>();
```
Comment (hdavidzhu) [inline app/src/main/java/boston/hackingforgood/resourceapp/components/home/HomeActivityFragment.java:34]: same here, move to separate method

File: app/src/main/java/boston/hackingforgood/resourceapp/components/home/RecyclerViewAdapter.java [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import java.util.List;
+
+import boston.hackingforgood.resourceapp.R;
+
+public class RecyclerViewAdapter extends RecyclerView.Adapter<RecyclerView.ViewHolder> {
```
Comment (hdavidzhu) [inline app/src/main/java/boston/hackingforgood/resourceapp/components/home/RecyclerViewAdapter.java:13]: more descriptive name, maybe?

File: app/src/main/java/boston/hackingforgood/resourceapp/components/home/ViewPagerAdapter.java [hunk 6]
```diff
@@ -0,0 +1,43 @@
+package boston.hackingforgood.resourceapp.components.home;
+
+import android.content.Context;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentPagerAdapter;
+
+import boston.hackingforgood.resourceapp.R;
+
+public class ViewPagerAdapter extends FragmentPagerAdapter {
```
Comment (hdavidzhu) [inline app/src/main/java/boston/hackingforgood/resourceapp/components/home/ViewPagerAdapter.java:10]: more descriptive name

File: app/src/main/res/layout/activity_home.xml [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+        android:layout_height="wrap_content"
+        android:text="Hello World!"
+        />
+
+
+    <android.support.v4.view.ViewPager
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:layout_weight="3"
+        android:id="@+id/viewPager"
```
Comment (hdavidzhu) [inline app/src/main/res/layout/activity_home.xml]: better naming

File: app/src/main/res/layout/activity_home.xml [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        android:layout_alignParentBottom="true"
+        >
+        <ImageButton
+            android:id="@+id/alertsButton"
+            android:layout_width="0dp"
+            android:layout_height="70dp"
```
Comment (hdavidzhu) [inline app/src/main/res/layout/activity_home.xml]: extract

File: app/src/main/res/layout/toolbar_layout.xml [hunk 9]
```diff
@@ -0,0 +1,11 @@
...
+<merge
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    >
+    <android.support.v7.widget.Toolbar
+        android:id="@+id/toolbar_main"
+        android:layout_width="match_parent"
+        android:layout_height="?attr/actionBarSize"
+        android:background="?attr/colorPrimary"
+        />
+</merge>
```
Comment (hdavidzhu) [inline app/src/main/res/layout/toolbar_layout.xml]: nice! - also you can rename this file to be `toolbar_main.xml` to match the toolbar id

File: app/src/main/java/boston/hackingforgood/resourceapp/components/home/HomeActivityFragment.java [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+
+import boston.hackingforgood.resourceapp.R;
+import butterknife.Bind;
+import butterknife.ButterKnife;
+
+public class HomeActivityFragment extends Fragment {
+    @Bind(R.id.recyclerView)
+    RecyclerView recyclerView;
+
+    public HomeActivityFragment() {
```
Comment (hdavidzhu) [inline app/src/main/java/boston/hackingforgood/resourceapp/components/home/HomeActivityFragment.java]: remove

File: app/src/main/res/layout/activity_home.xml (hunk 7 above)
Comment (MadelynTav) [inline app/src/main/res/layout/activity_home.xml]: alerted at changed PR @hdavidzhu

**Comment by hdavidzhu** [comment]
:ship:

**Pull request merged**